stroke-analysis/stroke_analysis.py

```diff
@@ -0,0 +1,32 @@
+# -*- coding: utf-8 -*-
+"""stroke_analysis.ipynb
+
+Automatically generated by Colab.
+
+Original file is located at
+    https://colab.research.google.com/drive/1m6TiVNZFLSE4gwaIKKeqhdvhR3JLl9oH
+"""
+
+import pandas as pd
+
+strokes= pd.read_csv("/content/healthcare-dataset-stroke-data.csv")
+
+strokes
+
+strokes.describe(include=float)
+
+strokes.info()
+
+strokeG = strokes.groupby("stroke")
+
+strokeG.get_group(1)
+
+strokes[strokes.stroke == 1].describe(include=float)
+
+strokes.describe(include=int)
```

Comment:
This should be called on strokeG.get_group(1) and describe should include another parameter to limit the percentile data